expression-tracker/lib/recognizer.rb

```diff
@@ -1,46 +1,54 @@
+# frozen_string_literal: true
+
+# Class for identifying OpenFace Action Units
 class Recognizer
-  TIME_INTERVAL=1.3
+  TIME_INTERVAL = 0.45
```

Comment:
Can you add a comment with the old value? What I have right now works pretty well, slightly concerned that an accidental recognition (like when I laugh or am surprised at something I read) will lead to a lot more false positive events than the current system.